README.md

```diff
@@ -127,6 +127,7 @@ scope of its wrapped input function.
 - [DragSelect](https://github.com/ThibaultJanBeyer/DragSelect): This plugin uses DragSelect to realize the "drag a rectangle to select multiple blocks" feature. The patching PR [#143](https://github.com/ThibaultJanBeyer/DragSelect/pull/143) and [#165](https://github.com/ThibaultJanBeyer/DragSelect/pull/165) made all this possible, and these PRs are included in [v2.6.0](https://github.com/ThibaultJanBeyer/DragSelect/releases/tag/v2.6.0).
 - [select.svg](test/media/select.svg) & [unselect.svg](test/media/unselect.svg): Free icons downloaded at [Icons8](https://icons8.com).
 - This plugin is part of the achievement by Songlin Jiang([@HollowMan6](https://github.com/HollowMan6)) participating the [Google Summer of Code 2022](https://summerofcode.withgoogle.com/programs/2022/projects/9wF06HWE) at [MIT App Inventor](https://github.com/mit-cml).
+- This plugin was updated for Blockly-v11 by Chang Min Bark([@changminbark](https://github.com/changminbark)) participating the [Google Summer of Code 2024](https://summerofcode.withgoogle.com/media/user/afbc2778524a/proposal/gAAAAABmfiJkQoI8AU7CQdVKcjTGzVcG_wTyrhZjekmhqyyZn88YsYUjdB_6gkAD-jYD5OoNDy09GXMQ4KPNVvqf2HEYk04J0KH4lWIn3iRY40eNEFr_GJw=.pdf) at [MIT App Inventor](https://github.com/mit-cml).
```

Comment:
I'll recommend you to use this link: https://summerofcode.withgoogle.com/archive/2024/projects/9916Xzin The link you are having here now doesn't work for everyone
![image](https://github.com/mit-cml/workspace-multiselect/assets/43995067/266f1005-e24e-4515-82e9-4c22135db386)